### PR TITLE
Upload transcript modal

### PR DESCRIPTION
**Pull request by KayleeLChan**
## Pull Request Summary

Response to review by Sam on https://github.com/BestFloww/BestFlow/pull/14 in new branch.

### Change types

- [x] Non-breaking change
- [x] Front end change

### CODE REVIEW

**Review by williamyao27 — APPROVED** [review]
Modal looks good and is functional (besides any centering we need)!

**Review by WeissSa — CHANGES_REQUESTED** [review]
A few small things, but this is well done! 👯 🥳 🔥

File: front-end/src/components/TranscriptUploadModal.jsx [hunk 1]
```diff
@@ -40,49 +33,25 @@ class TranscriptUploadModal extends React.Component {
...
-          right: '35vw',
-          bottom: '35vh',
-          border: '1px solid #ccc',
-          overflow: 'auto',
-          WebkitOverflowScrolling: 'touch',
-          borderRadius: '4px',
-          outline: 'none',
-          padding: '20px'
-        }
-      }}
```
[the remark this answers is not in the document]
Comment (KayleeLChan) [reply front-end/src/components/TranscriptUploadModal.jsx:67]: it's too spicy

**Comment by WeissSa** [comment]
Oh also we can uncomment the lines regarding the TranscriptService! I have a PR up now.

File: front-end/src/components/TranscriptUploadModal.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
-import React, { Component } from 'react';
+import React from 'react';
 import axios from 'axios';
 import Modal from 'react-modal';
+//import TranscriptAPI from "./services/TranscriptAPI";
```
Comment (WeissSa) [inline front-end/src/components/TranscriptUploadModal.jsx:4]:
```suggestion
import TranscriptAPI from "../services/TranscriptAPI.js";
```

**Review by WeissSa — APPROVED** [review]
Almost there! Just implement the 2 small changes we discussed on discord and this is good to merge.

**Pull request merged**